### ISSUE TITLE
Multiprocessing for CPU portion of models.Cellpose.eval?

### ISSUE THREAD

**Issue by aravindkoti** [MouseLand/cellpose#850]
Hi Cellpose team, 

I am running a 3D segmentation job and noticed that during my run, the GPU is used heavily for the first 5-10 minutes to find all the masks when I run "model.eval()". Then all the computation is switched over to the CPU for the rest of the run, and the total run time is ~4hrs. Seeing as how the CPU is used during most of the process, I would like to multiprocess this part of the segmentation since I'm not taking advantage of all my cores - is there a way to do this?

Thank you!

**Comment by mrariden** [comment]
@aravindkoti Multiprocessing is not built into the `.eval()` methods. It might be possible to deconstruct the `CellposeModel.eval()` method and implement multiprocessing, but this not supported by us. 

Ongoing efforts are to identify inefficiencies and improve throughput. But due to operating system-dependent behavior, multiprocessing isn't an ideal option. Unfortunately, Windows doesn't play well with it.

**Comment by carsen-stringer** [comment]
we are working on this to parallelize mask dynamics on the GPU for large images, going to close this issue for now as it will be a different solution
